### ISSUE TITLE
PushBullet invalid syntax error

### ISSUE THREAD

**Issue by tam481** [jpmens/mqttwarn#259]
Hello,
Newbie here. I'm trying to configure my MQTTWarn to send alerts via PushBullet and got stuck trying to get it to work. I get the following error message logged in the log file:

ERROR [mqttwarn] Can't load pushbullet service (services/pushbullet.py): invalid syntax (pushbullet.py, line 7)

**Comment by jpmens** [comment]
What platform are you running on, which version of Python, and which steps have you done to get that error? There is nothing on line 7 ...

**Comment by jpmens** [comment]
Lack of response.

**Comment by tam481** [comment]
My apologies. Just last night I managed to figure out where I was going wrong. I had downloaded the pushbullet.py file using wget and it downloaded the "html" from the repository for some reason. I copied and pasted the raw text into nano and it complained about something else which I believe is related to my version of python. I have versions 2.7 and 3.x installed and I'm struggling to get it to work. Many thanks for taking the time to look at the issue and I'm sorry for not replying sooner. For some reason I did not receive the notification email I usually receive.
